From 87d8f8e666b91e2bc63f3c24212b7ad4253e1458 Mon Sep 17 00:00:00 2001
From: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date: Fri, 17 Aug 2018 13:12:00 +0000
Subject: [PATCH] OS-6913 ubuntu 18.04 hangs during boot on bhyve Reviewed by:
 Mike Gerdts <mike.gerdts@joyent.com> Reviewed by: John Levon
 <john.levon@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 .../generic/usr/share/bhyve/uefi-csm-rom.bin  | Bin 1048576 -> 1048576 bytes
 overlay/generic/usr/share/bhyve/uefi-rom.bin  | Bin 1048576 -> 1048576 bytes
 2 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100644 => 100755 overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
 mode change 100644 => 100755 overlay/generic/usr/share/bhyve/uefi-rom.bin

diff --git a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin
old mode 100644
new mode 100755
index 0687c892..a90f7760
Binary files a/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-csm-rom.bin differ
diff --git a/overlay/generic/usr/share/bhyve/uefi-rom.bin b/overlay/generic/usr/share/bhyve/uefi-rom.bin
old mode 100644
new mode 100755
index cfefe0be..ba2214a4
Binary files a/overlay/generic/usr/share/bhyve/uefi-rom.bin and b/overlay/generic/usr/share/bhyve/uefi-rom.bin differ
-- 
2.21.0

